.travis.yml

```diff
@@ -13,17 +13,17 @@ git:
 matrix:
   fast_finish: true
   allow_failures:
-    - julia: 1.2  # development versions
+    - julia: 1.3  # development versions
       env: TEST_SUITE='using Pkg; Pkg.develop("LazySets"); Pkg.develop("MathematicalSystems"); Pkg.develop("HybridSystems"); Pkg.develop("MathematicalPredicates"); Pkg.develop("TaylorModels"); Pkg.build("Reachability"); Pkg.test("Reachability"; coverage=true)'; DOCS=false
-  include:
-    - julia: 1.0  # master versions
-      env: TEST_SUITE='using Pkg; Pkg.build("Reachability"); Pkg.test("Reachability"; coverage=true)'; DOCS=false
-    - julia: 1.1  # master versions
-      env: TEST_SUITE='using Pkg; Pkg.build("Reachability"); Pkg.test("Reachability"; coverage=true)'; DOCS=false
-    - julia: 1.2  # master versions
+  #include:
+    #- julia: 1.0 
+    #  env: TEST_SUITE='using Pkg; Pkg.build("Reachability"); Pkg.test("Reachability"; coverage=true)'; DOCS=false
+    #- julia: 1.1 
+    #  env: TEST_SUITE='using Pkg; Pkg.build("Reachability"); Pkg.test("Reachability"; coverage=true)'; DOCS=false
```

Comment:
You completely deactivated those runs. Instead I would copy them to `allow_failures`.

Reply:
:+1:  now i left v1.3 with the dev versions and the rest with the released versions.

but feel free to change the script to your convenience!

Reply:
in my machine i use v1.4 and it works as well, i dunno if we should add this already

Reply:
You have to *copy*, not move. The `allow_failures` entry must still stay in the `include` part.

Reply:
> in my machine i use v1.4 and it works as well, i dunno if we should add this already

If v1.4 works here, sure (but then also update `Project`).